FlowCryptUITests/SignInImapTest.swift

```diff
@@ -43,6 +40,102 @@ extension SignInImapTest {
         wait(4)
         XCTAssert(app.buttons["+"].exists)
     }
+    
+    // restart app -> loads inbox
+    func test_2_restart_app_load_inbox() {
+        let application = XCUIApplication()
+        wait(2)
+        application.buttons["+"].tap()
+        
+        wait(0.3)
+        
+        application.typeText("ElonMusk@gmail.com")
+        application.tables.textFields["Subject"].tap()
+        wait(3)
+        application.tables.textFields["Subject"].tap()
+        
+        application.typeText("SpaceX")
+        
+        snapshot("compose")
+        application.navigationBars.buttons["arrow left c"].tap()
+        wait(1)
+        
+        tapOnCell()
+        snapshot("message")
+        app.navigationBars.buttons["arrow left c"].tap()
+        
+        wait(1)
+        
+        menuButton.tap()
+        snapshot("menu")
+        
+        tapOnMenu(folder: "Settings")
+        snapshot("settings")
+        
+        // TODO: - https://github.com/FlowCrypt/flowcrypt-ios/issues/470
+        /*
+         let app = XCUIApplication()
+         let inboxNavigationBar = app.navigationBars["Inbox"]
+         inboxNavigationBar.buttons["help icn"].tap()
+         inboxNavigationBar.buttons["search icn"].tap()
+         app.otherElements["PopoverDismissRegion"].tap()
+         app.navigationBars["Search"].buttons["arrow left c"].tap()
+         
+         let tablesQuery2 = app.tables
+         let cellsQuery = tablesQuery2.cells
+         cellsQuery.otherElements.containing(.staticText, identifier:"Jun 07").staticTexts["denbond7@flowcrypt.test"].tap()
+         tablesQuery2.children(matching: .cell).element(boundBy: 2).children(matching: .other).element.children(matching: .other).element.children(matching: .textView).element.children(matching: .textView)["It's an encrypted text"].tap()
+         inboxNavigationBar.buttons["arrow left c"].tap()
+         
+         let tablesQuery = tablesQuery2
+         tablesQuery.staticTexts["..."].tap()
+         
+         //
+         cellsQuery.otherElements.containing(.staticText, identifier:"encrypted message with missing key error").staticTexts["denbond7@flowcrypt.test"].tap()
+         tablesQuery.textViews.textViews["Could not decrypt:"].tap()*/
+    }
+    
+    // login -> cancel
+    func test_4_login_cancel() {
+        let user = UserCredentials.imapDev
+        loginWithImap(user)
+
+        passPhraseTextField.swipeUp()
+        tapUseAnotherAccountAndVerify()
+    }
+    
+    // login with user without key backups and emails
+    // login -> no messages
+    func test_5_login_no_messages() {
+        verifyFlowWithNoBackups(for: .imapDen)
+    }
+
+    func test_6_has_msgs_no_backups() {
+        verifyFlowWithNoBackups(for: .imapHasMessagesNoBackups)
+        
+        
+                
+    }
+    
+    // login with wrong pass phrase
+    func test_7_login_bad_pass_phrase() {
+        let user = UserCredentials.imapDev
+        loginWithImap(user)
+        
+        
```

Comment:
Is there a way to set maximum one empty line in our linter?

Reply:
We have this autocorrect rule, need to apply it to test target as well

Reply:
created a separate issue for this